src/components/Sidebar/SearchResults.js

```diff
@@ -39,15 +39,15 @@ function SearchResults({
   }
   if (searchResults.length === 0 && currentSearchInput.length >= 3) {
     return (
-      <Typography className={classes.inputHint} variant="body1">
+      <Typography className={classes.inputHint}>
         No results
       </Typography>
     );
   }
   return (
     <div className={classes.searchResults}>
       {currentSearchInput.length > 0 && currentSearchInput.length <= 2
-        && <Typography className={classes.inputHint} variant="body1"> Keep typing! </Typography>
+        && <Typography className={classes.inputHint}> Keep typing! </Typography>
```

Comment:
Nitpick, but we don't need space on either side of "Keep typing!" - I know that's not a part of the changes you made here but might as well roll it in with this PR.